### ISSUE TITLE
lost some file in training.

### ISSUE THREAD

**Issue by liulipeng8023** [biansy000/ShapeBoost#7]
wonderful work for 3D HPS!
When I reproduced the training, I needed some pre-trained weights in the shapy loss, but these files were not found in the shapy downloadable files, such as “width_bonelen2measurements_2layers_48.pth” and “caesar-female_smplx-neutral-10betas/last_simple.ckpt”

**Comment by biansy000** [comment]
@liulipeng8023  Thanks for your interest in our work! However, I believe these files are open-sourced by [Shapy](https://shapy.is.tue.mpg.de/index.html), and due to licensing restrictions, I cannot share them directly.

**Comment by liulipeng8023** [comment]
Thank you for your reply. I will check it again.

**Comment by liulipeng8023** [comment]
I double-checked in shapy and couldn't find the model file “width_bonelen2measurements_2layers_48.pth.” I also took a closer look at your training code. This file is used to estimate body measurements (chest, hip, and waist circumference) using body part width and length through this model. So, I'd like to confirm with you again if this model file is the one you trained?
 Thank you.

**Comment by biansy000** [comment]
@liulipeng8023 I double-checked the files, and you are right - this one was missing. I have uploaded it to the repo, so you can try running our code again.

**Comment by liulipeng8023** [comment]
Thanks for your reply. I'll try again

**Comment by liulipeng8023** [comment]
Hi, siyuan:
I note that  other files are also missing, such as “caesar-male_smplx-neutral-10betas/last_simple.ckpt” and “caesar-female_smplx-neutral-10betas/last_simple.ckpt”. Under the shapy weights directory, I found “caesar-male_smplx-neutral-10betas/last.ckpt”  and “caesar-female_smplx-neutral-10betas/last.ckpt” under the shapy weights directory, but these weights do not match the model structure you defined and will result in loading.@biansy000

**Comment by biansy000** [comment]
@liulipeng8023 Sorry! You can use the weights [here](https://drive.google.com/drive/folders/1I1Amqzn1eqV-PFIbD5q3_qzyZs3zCX9C?usp=drive_link)

**Comment by liulipeng8023** [comment]
Thanks, I've submitted a request for download permissions to this cloud storage. Please grant the authorization. @biansy000

**Comment by biansy000** [comment]
@liulipeng8023 Done.

**Comment by liulipeng8023** [comment]
@biansy000 Thanks.
